### ISSUE TITLE
Chaining method calls often doesn't work (count, pluck, ...)

### ISSUE THREAD

**Issue by bronson** [Casecommons/pg_search#226]
The docs imply that chains [just work](https://github.com/Casecommons/pg_search/blob/ad5d773e246dd12c904fdcc91e343739dfcec0fe/README.md#chaining-method-calls-onto-the-results), but the first two chains I tried failed:

    PgSearch.multisearch('dock').count
    : SELECT COUNT("pg_search_documents".*, ((ts_rank((to_tsvector('english', unaccent(coalesce("pg_search_documents"."content"::text, '')))), (to_tsquery('english', ''' ' || unaccent('dock') || ' ''')), 0))) AS pg_search_rank) FROM "pg_search_documents"  WHERE (((to_tsvector('english', unaccent(coalesce("pg_search_documents"."content"::text, '')))) @@ (to_tsquery('english', ''' ' || unaccent('dock') || ' '''))))
    ActiveRecord::StatementInvalid: PG::SyntaxError: ERROR:  syntax error at or near "AS"
    LINE 1: ...lish', ''' ' || unaccent('dock') || ' ''')), 0))) AS pg_sear...
                                                             ^

    PgSearch.multisearch('dock').pluck(:searchable_id)
    : SELECT "pg_search_documents"."searchable_id" FROM "pg_search_documents"  WHERE (((to_tsvector('english', unaccent(coalesce("pg_search_documents"."content"::text, '')))) @@ (to_tsquery('english', ''' ' || unaccent('dock') || ' '''))))  ORDER BY pg_search_rank DESC, "pg_search_documents"."id" ASC
    ActiveRecord::StatementInvalid: PG::UndefinedColumn: ERROR:  column "pg_search_rank" does not exist
    LINE 1: ... ''' ' || unaccent('dock') || ' '''))))  ORDER BY pg_search_...
                                                             ^

Are the docs incomplete, or am I doing something wrong?

**Comment by bronson** [comment]
A workaround for pluck, specify `reorder('')`:
```ruby
PgSearch.multisearch('dock').reorder('').pluck(:searchable_id)
```

**Comment by nertzy** [comment]
I believe this is fixed by eb730f705c5c532a7e26b496392b7e09e0b8d713, which I just pushed to master.

I added some tests in a074acbd99da94699be061bf796b6bcb23202a8a, 835143f800b9f877fd06cb475f2796a8093d3c92, and 1df7c21632d418547a02034ca6cd14f523472922

Can you confirm?
